packages/analytics-nextjs/src/context.tsx

```diff
@@ -139,11 +151,12 @@ export function AnalyticsContextProvider({
                             domain: document.location.host,
                             ...cookie,
                         },
+                        integrations,
                         user,
                         // Disable calls to Segment API completely if no Write Key is provided
                         ...(!writeKey && {
                             // eslint-disable-next-line @typescript-eslint/naming-convention
-                            integrations: { 'Segment.io': false },
+                            integrations: { ...integrations, 'Segment.io': false },
```

Comment:
I've added it here just in case someone wants to use `integrations` but not provide `writeKey` 🤔 Although I don't know if it makes sense without `writeKey` at all? 🤔

Reply:
I don't think Integrations will work without a Segment write key. `Segment.io` is the base integration that provides the core tracking functionality, and I would assume that all other integrations depend on it to function. Not much said about this in the docs though 🤔

Reply:
Alright, I'll revert it 👍